Makefile

```diff
@@ -60,5 +60,6 @@ systemtests:
 
 client_install:
 	npm install -g cli-rhea
+	pip install cli-proton-python
```

Comment:
make sure that pip is installed in system and add it into the list of prerequisites: https://github.com/EnMasseProject/enmasse/blob/master/HACKING.md